pages/robots/rbts-raspberry-pi.md

```diff
@@ -36,6 +36,40 @@ So far we are working on creating a seamless experience for the image, meaning w
 
 There is always something to be done. Check the [GitHub issues](https://github.com/ole-vi/treehouse-builder/issues) for treehouse-builder to see what needs to be done or [waffle.io](https://waffle.io/ole-vi/treehouse-builder). Once you've chosen an issue and fixed the problem, create a PR using the same guidelines as the ones used when you had to go through the intern orientation. If you have any questions, just ask! You can find us on the #raspberrypi channel on gitter.
 
+Since we are customizing the image, we would like to access it without using password everytime. In order to do that,[our implementation](https://github.com/ole-vi/treehouse-builder/blob/master/get_ssh_keys.py) requires you to have a [SSH key in your GitHub account](https://github.com/settings/keys). If you don't have one for the computer your are working on, please follow [the guide on GitHub Help](https://help.github.com/articles/connecting-to-github-with-ssh/) and return here when you are finished.
+
+After you are done generating and adding SSH keys, clone the treehouse builder with SSH by following the command,
+```sh
+git clone git@github.com:ole-vi/treehouse-builder.git
+```
+
+After you are done cloning, go to the folder where treehouse-builder is located and create a new branch as you created previously and push.
```

Comment:
Could you explain why we are having this step?

Reply:
Should I add something like this "Creating a new branch allows us to make changes to the repository in a safe manner where if something goes wrong, we can always revert back to the master branch and start anew"  ?? @xyb994

Reply:
@lorugant Sorry about the confusion, my question was why is this section here. I thought this PR is about explaining how to ssh without using password.

Reply:
@xyb994 I was told to write about how to cross verify the two keys by @dogi